LX.PersonalDevelopmentPlan.Tests/TestSettings.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Globalization;
+
+namespace LX.PersonalDevelopmentPlan.Tests;
+
+public partial class DateTimeParsingTests
+{
+    public static class TestSettings
```

Comment:
this could be standalone static class and not a part of partial of DateTimeParsingTests